### PR TITLE
[Papercut][SW-14407] update button for product feeds

### PR DESCRIPTION
**Pull request by mkreusch**
## Description

Please describe your pull request:
- Why is it necessary?
  Papercut SW-14407 - sometimes you do not want the product feed window to be closed on saving, e.g. when you are trying different settings
- What does it improve?
  see above
- Does it have side effects?
  no

| Questions | Answers |
| --- | --- |
| BC breaks? | no |
| Tests pass? | yes |
| Related tickets? | https://issues.shopware.com/#/issues/SW-14407 |
| How to test? | Open a product feed and use the new "Update"/"Aktualisieren" button |

### CODE REVIEW

File: themes/Backend/ExtJs/backend/product_feed/controller/feed.js [hunk 1]
```diff
@@ -332,7 +347,9 @@ Ext.define('Shopware.apps.ProductFeed.controller.Feed', {
                     var data = response.data;
                     attributeForm.saveAttribute(data.id);
                     Shopware.Notification.createGrowlMessage('',me.snippets.onSaveChangesSuccess, me.snippets.growlMessage);
-                    me.getProductFeedWindow().destroy();
+                    if(typeof(keepOpened) == 'undefined' || !keepOpened){
```
Comment (mitelg) [inline themes/Backend/ExtJs/backend/product_feed/controller/feed.js]: you can use the `Ext.isDefined` function here

http://docs.sencha.com/extjs/4.1.1/#!/api/Ext-method-isDefined

File: themes/Backend/ExtJs/backend/product_feed/controller/feed.js [hunk 2]
```diff
@@ -303,7 +318,7 @@ Ext.define('Shopware.apps.ProductFeed.controller.Feed', {
      * @param [object] btn - pressed Ext.button.Button
      * @return void
      */
-    onSave: function (btn) {
+    onSave: function (btn, keepOpened) {
```
Comment (mitelg) [inline themes/Backend/ExtJs/backend/product_feed/controller/feed.js]: parameter doc is missing

**Comment by mkreusch** [comment]
@mitelg Thanks for the valuable advice - i have corrected this

**Comment by bcremer** [comment]
@mkreusch For you information, I changed the wording a bit to be more consistent with some existing modules: https://github.com/shopware/shopware/commit/042cedb8a6ad4311fd4d1085f0af1d1d5ad00931

**Pull request merged**